hphp/hack/src/heap/hh_shared.c

```diff
@@ -142,11 +142,11 @@
 #if !defined __APPLE__ && !defined _WIN32
   // Linux version for the architecture must support syscall memfd_create
```

Comment:
I'm fine with defining it when it isn't defined, but if it's already defined then we should be relying on it.